charts/redpanda/values.go

```diff
@@ -1497,8 +1498,16 @@ func (l *KafkaListeners) TrustStores(tls *TLS) []*TrustStore {
 	return tss
 }
 
-func (k *KafkaListeners) ConsolemTLS(tls *TLS) config.KafkaTLS {
```

Comment:
I never changed this because I thought it was "Console mTLS" 😬

Reply:
It can be mutual TLS if `TLS.RequireClientAuth` is set to `true`